hudi-utilities/src/main/java/org/apache/hudi/utilities/HoodieClusteringJob.java

```diff
@@ -49,6 +51,9 @@
   private transient FileSystem fs;
   private TypedProperties props;
   private final JavaSparkContext jsc;
+  private static final String EXECUTE = "execute";
+  private static final String SCHEDULE = "schedule";
+  private static final String SCHEDULE_AND_EXECUTE = "scheduleandexecute";
```

Comment:
is scheduleandexecute -> scheduleAndExecute better?

Reply:
emmmm, reasons for using all lowercase is that we use `switch (cfg.runningMode.toLowerCase()) {xxx}` to do switch, so that users can use --mode scheduleAndExecute/scehduleANDEXECUTE/SCHEDULEandEXECUTE, etc

Reply:
is remove toLowerCase better, it can reduce user's Confused